web/gds-user-ui/src/components/RegistrationForm/InvalidFormPrompt.tsx

```diff
@@ -44,11 +45,12 @@ function InvalidFormPrompt({
               <Text>
                 <Trans>
                   If you continue, your changes will be lost. To save your changes, click Cancel and
-                  then click on :
+                  then click on the
                 </Trans>
               </Text>
               <Text fontWeight={'bold'} as="span">
-                <Trans>{isNextStep ? 'Save & Next' : 'Save & Previous'} button.</Trans>
+                {isNextStep ? t`Save & Next` : t`Save & Previous`}{' '}
```

Comment:
I couldn't figure out why `0` was returning, so let's try removing the conditional from the `<Trans>` tags.

Reply:
I can see what could happen. 

const [btnContent, setIsBtnContent] = useState<string>("")

UseEffect(()=> {
  If(isNextStep)

   SetBtnContent('btn content here');

  } else {

    SetBtnContent('btn content 2 here');

  }


}, [isNextStep])

 And use btnContent to display the button content